README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
 Repository for homework 01
+
+#**Sarah Jensen**
+##Intro
+###My name is Sarah, and I am a 2018 graduate of St. Norbert College where I earned my B.A. in Psychology and French. I study moral psychology and am interested in how children develop their sense of right and wrong based on their social environments. 
```

Comment:
Did you intend for this entire line to be a header itself? If not, you needed a break after the portion that you wanted as the header. If so, this is way too much material to be used as a "header" which should be just an organizing phrase. If you were looking to make the font larger you should have picked another formatting option, like **bold**.